spec/models/export_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "rails_helper"
+
+RSpec.describe Export do
+  describe ".enqueue_faxes" do
+    it "schedules jobs for signed, unfaxed application updated awhile ago" do
+      signed_unfaxed_updated_awhile_ago = create(:snap_application,
+                                                 signed_at: 31.minutes.ago,
```

Comment:
`31` is referenced a lot here and feels like a magic number. What about referencing a constant instead?